spec/features/state_file/review_page_spec.rb

```diff
@@ -335,6 +335,102 @@
+            expect(page).to have_text "$250.00"
+
+            click_on I18n.t("general.review_and_edit")
+          end
+
+          # first eligible 1099R
+          expect(page).to have_text I18n.t("state_file.questions.id_retirement_and_pension_income.edit.subtitle")
+          expect(page).to have_text("Dorothy Red")
+          expect(page).to have_text("$200")
+          click_on I18n.t("general.continue")
+
+          # second eligible 1099R
+          expect(page).to have_text I18n.t("state_file.questions.id_retirement_and_pension_income.edit.subtitle")
+          expect(page).to have_text("Couch Potato Cafe")
+          expect(page).to have_text("$50")
+          choose "No"
+          click_on I18n.t("general.continue")
+
+          expect(page).to have_text I18n.t("state_file.questions.shared.abstract_review_header.title")
+          within "#qualified-retirement-benefits-deduction" do
+            expect(page).to have_text I18n.t("state_file.questions.id_review.edit.qualified_retirement_benefits_deduction")
+            expect(page).to have_text I18n.t("state_file.questions.id_review.edit.qualified_retirement_benefits_deduction_explain")
+            expect(page).to have_text I18n.t("state_file.questions.id_review.edit.qualified_disabled_retirement_benefits")
+            expect(page).to have_text "$200.00" # $50 less eligible
+          end
+        end
+      end
+
+      context "with eligible disabled senior under 65 years old" do
+        before do
+          @intake.update(primary_birth_date: Date.new((MultiTenantService.statefile.current_tax_year - 64), 12, 31))
+        end
+
+        it "review & edit questions on disability, and skip eligible income question, then return to review & does not see the card (no longer eligible)" do
+          visit "/questions/id-review"
+
+          expect(page).to have_text I18n.t("state_file.questions.shared.abstract_review_header.title")
+          within "#qualified-retirement-benefits-deduction" do
+            expect(page).to have_text I18n.t("state_file.questions.id_review.edit.qualified_retirement_benefits_deduction")
+            expect(page).to have_text I18n.t("state_file.questions.id_review.edit.qualified_retirement_benefits_deduction_explain")
+            expect(page).to have_text I18n.t("state_file.questions.id_review.edit.qualified_disabled_retirement_benefits")
+            expect(page).to have_text "$250.00"
+
+            click_on I18n.t("general.review_and_edit")
+          end
+
+          expect(page).to have_text I18n.t("state_file.questions.id_disability.edit.title")
+          choose "No"
+          click_on I18n.t("general.continue")
+
+          expect(page).to have_text I18n.t("state_file.questions.shared.abstract_review_header.title")
+          expect(page).not_to have_text I18n.t("state_file.questions.id_review.edit.qualified_retirement_benefits_deduction")
```

Comment:
This spec tests the destroying of followups in `IdDisabilityForm` if the user chooses to select "No" for disability. They already have a followup (created at the top of the spec) which should be destroyed -- leading to the calculations for the review card to **not** sum up the value of the 1099Rs (both are primary's)